src/site/stages/build/process-cms-exports/transformers/paragraph-media.js

```diff
@@ -9,7 +9,7 @@ const transform = entity => ({
       getDrupalValue(entity.fieldAllowClicksOnThisImage) || false,
     fieldMedia:
       entity.fieldMedia && entity.fieldMedia.length
-        ? { entity: getImageCrop(entity.fieldMedia[0], '_21MEDIUMTHUMBNAIL') }
```

Comment:
Looking at the [graphQL query](https://github.com/department-of-veterans-affairs/vets-website/blob/master/src/site/stages/build/drupal/graphql/paragraph-fragments/media.paragraph.graphql.js), I don't see that it's needed